forge-gui/res/cardsfolder/upcoming/young_deathclaws.txt

```diff
@@ -0,0 +1,7 @@
+Name:Young Deathclaws
+ManaCost:2 B G
+Types:Creature Lizard Mutant
+PT:4/2
+K:Menace
+S:Mode$ Continuous | Affected$ Creature.YouOwn | AffectedZone$ Graveyard | AddKeyword$ Scavenge:CardManaCost | Description$ Each creature card in your graveyard has scavenge. The scavenge cost is equal to its mana cost.
```

Comment:
Maybe deckhints: Ability$Graveyard|Mill

Reply:
Oh, sure thing :) While I was at it, I had a look at other scavenge cards and lifted pointers from them in this regard.